src/functions/images.js

```diff
@@ -0,0 +1,16 @@
+const { Function } = require("@kcp/functions");
+const { get } = require("snekfetch");
```

Comment:
Should be using node-fetch so we can slowly move away from snekfetch the deprecated module.